language/stdlib/modules/libra_block.move

```diff
@@ -19,7 +19,7 @@ module LibraBlock {
     struct NewBlockEvent {
       round: u64,
       proposer: address,
-      previous_block_votes: vector<u8>,
+      previous_block_votes: vector<address>,
```

Comment:
this is interesting... is it tested anywhere? forgive my naiveté.

Reply:
what is tested? this event or this field?